src/aui-form-validator/js/aui-form-validator.js

```diff
@@ -588,6 +588,12 @@ var FormValidator = A.Component.create({
 
 			removeClasses(field, [VALID_CLASS, ERROR_CLASS]);
 			removeClasses(fieldContainer, [CONTAINER_VALID_CLASS, CONTAINER_ERROR_CLASS]);
+
+			if (field.get(TYPE).toLowerCase() == RADIO && fieldContainer) {
+				var radioFieldContainer = fieldContainer.ancestor('.aui-field-wrapper-content');
+
+				removeClasses(radioFieldContainer, [CONTAINER_VALID_CLASS, CONTAINER_ERROR_CLASS]);
+			}
```

Comment:
I tried reverting LPS-47418, but when I did so I saw the same problem as before where the validator warning sign only appears next to the first radio button.

![image](https://cloud.githubusercontent.com/assets/15066512/17181595/8fc1fc0a-53d5-11e6-92c7-c99f6bbf07b3.png)